Doc/library/typing.rst

```diff
@@ -592,7 +592,7 @@ The module defines the following classes, functions and decorators.
    when the checked program targets Python 3.9 or newer.
 
    The deprecated types will be removed from the :mod:`typing` module
-   in the first Python version released 5 years after the release of Python 3.9.0.
+   no sooner than the first Python version released 5 years after the release of Python 3.9.0.
```

Comment:
This would be 3.13 I believe, and I don't think we're going to remove them in 3.13.